webapp/channels/src/packages/mattermost-redux/src/actions/admin.ts

```diff
@@ -61,6 +61,13 @@ export function getPlainLogs(page = 0, perPage: number = General.LOGS_PAGE_SIZE_
     });
 }
 
+export function getAllPlainLogs() {
+    return bindClientFunc({
+        clientFunc: Client4.getAllPlainLogs,
+        onSuccess: [AdminTypes.RECEIVED_ALL_PLAIN_LOGS],
```

Comment:
Again, assuming we don't move more of this to the server, I'd rather not store this data in redux state since it's only used once. If you remove the `onSuccess` argument from here, it won't save anything in Redux, but you can still get the result by from the return value from the action which would be an `ActionResult<string[]>`